dropwizard-migrations/src/main/java/io/dropwizard/migrations/CloseableLiquibase.java

```diff
@@ -11,8 +11,8 @@
 public abstract class CloseableLiquibase extends Liquibase implements AutoCloseable {
     private final ManagedDataSource dataSource;
 
-    public CloseableLiquibase(String changeLogFile, ResourceAccessor resourceAccessor, DatabaseConnection conn, ManagedDataSource dataSource) throws LiquibaseException, SQLException {
-        super(changeLogFile, resourceAccessor, conn);
+    public CloseableLiquibase(String changeLogFile, ResourceAccessor resourceAccessor, Database database, ManagedDataSource dataSource) throws LiquibaseException, SQLException {
```

Comment:
Thanks for the feedback.  I could overload the constructor, keeping the old one in-tact, to avoid this. What do you think?